code/testsuite/csheets/pf_Cleric.xml

```diff
@@ -1701,14 +1701,14 @@ BIO
 				<bonuslist></bonuslist>
 			</item>
 			<item>
-				<name>Blanket (Winter)</name>
+				<name>Blanket</name>
```

Comment:
Uh, Winter Blanket becomes Blanket?

Reply:
It's listed as "winter blanket" in the CR as a leftover from 3.5, then there's a lighter weight and cheaper cost item simply called "blanket" in the APG. Then the item "blanket" is listed in UE with the original weight and cost as the "winter blanket" with the description from the APG.

Reply:
Okay, and is this test unit pulling from the CR, the APG or the UE? If it's the CR, and the UE isn't loaded, then the CR name should be prominent.

Reply:
Well the cleric test loads both Core Rulebook and Advanced Player's Guide. Which version is it supposed to have?

Reply:
Also, it's a pain to nearly impossible to change the display name. There really should be a MOD tag to change it.